src/main/java/org/broadinstitute/consent/http/resources/DatasetResource.java

```diff
@@ -314,15 +315,11 @@ public Response updateDataset(@Auth AuthUser authUser, @Context UriInfo info,
   @Produces("application/json")
   @PermitAll
   @Path("/v2")
-  @Deprecated
-  public Response findAllDatasetsAvailableToUser(@Auth AuthUser authUser,
-      @QueryParam("asCustodian") Boolean asCustodian) {
```

Comment:
The `asCustodian` feature was an early experiment and never actually used in practice.